apps/web/lib/api/apiConstants.ts

```diff
@@ -10,3 +10,4 @@ export const API = {
 };
 // export const FILTERING = "http://localhost:3000/api/map/filtering/layer/";
 export const FILTERING = (layer_id: string) => `https://geoapi.goat.dev.plan4better.de/collections/${layer_id}/tiles/{z}/{x}/{y}`;
+export const GEO_API_ROOT = "https://geoapi.goat.dev.plan4better.de"
```

Comment:
We shouldn't hardcode this in constants as it should come from env variables as the url is different for prod, staging, dev etc. . 
This can already be accessed from process.env.NEXT_PUBLIC_GEOAPI_URL. Same with filtering.